modules/ztp-provisioning-lvm-storage.adoc

```diff
@@ -37,13 +39,31 @@ You use {lvms} to persist event subscriptions when you use PTP events or bare-me
     channel: stable-{product-version}
   policyName: subscription-policies
 ----
++
+[NOTE]
+====
+The Storage LVMO subscription is deprecated. In future releases of {product-title}, the storage LVMO subscription will not be available. Instead, you must use the Storage LVMS subscription. 
+
+In {product-title} {product-version}, you can use the Storage LVMS subscription instead of the LVMO subscription. The LVMS subscription does not require manual overrides in the `common-ranGen.yaml` file. Add the following YAML to `spec.sourceFiles` in the `common-ranGen.yaml` file to use the Storage LVMS subscription:
```

Comment:
Would this be like a YAML snippet? Not sure. Just wondering. 

```suggestion
In {product-title} {product-version}, you can use the Storage LVMS subscription instead of the LVMO subscription. The LVMS subscription does not require manual overrides in the `common-ranGen.yaml` file. Add the following YAML snippet to `spec.sourceFiles` in the `common-ranGen.yaml` file to use the Storage LVMS subscription:
```